go/libraries/doltcore/sqle/enginetest/dolt_server_test.go

```diff
@@ -235,11 +235,11 @@ var DoltBranchMultiSessionScriptTests = []queries.ScriptTest{
 			},
 			{
 				Query:          "/* client a */ select name from dolt_branches;",
-				ExpectedErrStr: "Error 1105: current branch has been force deleted. run 'USE <database>/<branch>' to checkout a different branch, or reconnect to the server",
+				ExpectedErrStr: "Error 1105: database not found: dolt/branch1",
```

Comment:
Yeah, this error was coming from a very specific code path that got altered beyond recognition, agree it would have been nice to save but it's such an edge case